### DISCUSSION TITLE
Server "cpplint" is not a valid entry in ensure_installed

### DISCUSSION THREAD

**Discussion by ghost** [mason-org/mason.nvim#1162] [Q&A] [answered]
<img width="947" alt="1" src="https://user-images.githubusercontent.com/50235281/229269183-1c437501-f65a-4c8f-8fae-635792ecdfe9.png">
<img width="167" alt="2" src="https://user-images.githubusercontent.com/50235281/229269195-c13d07a1-c5c4-44c0-9fe1-f1ac27c14e99.png">

Can not install cpplint package via lsp.ensure_installed(..). How to configure so that cpplint installs automatically along with clangd?

**Comment by yutkat** [comment, marked as answer]
cpplint is Linter, it's not LSP so it won't work with that. null-ls.nvim(https://github.com/jose-elias-alvarez/null-ls.nvim) or something like that would work.

**Reply by ghost** [reply]
Thanks.. It worked using null-ls.
